pulp_smash/tests/pulp3/constants.py

```diff
@@ -22,17 +22,17 @@
 FILE_PUBLISHER_PATH = urljoin(BASE_PUBLISHER_PATH, 'file/')
 
 IMPORTER_DOWN_POLICY = {'background', 'immediate', 'on_demand'}
-"""Download policies for an importer.
+"""Download policies for an remote.
 
-See `pulpcore.app.models.Importer
-<https://docs.pulpproject.org/en/3.0/nightly/contributing/platform_api/app/models.html#pulpcore.app.models.Importer>`_.
+See `pulpcore.app.models.Remote
+<https://docs.pulpproject.org/en/3.0/nightly/contributing/platform_api/app/models.html#pulpcore.app.models.Remote>`_.
 """
 
 IMPORTER_SYNC_MODE = {'mirror'}
```

Comment:
Same. I'm OK with leaving this name as is, so long as an issue is created in Smash's issue tracker.